rhobs/alerting/data_plane/prometheus.release_service_alerts.yaml

```diff
@@ -83,3 +83,18 @@ spec:
         alert_team_handle: <!subteam^S03SVBS426R>
         runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/release-service/release-controller-availability.md?ref_type=heads
         team: release
+
+    - alert: ReleaseServiceControllerManagerPodNotReady
+      expr: |
+        sum by (source_cluster, namespace) (kube_pod_container_status_ready{container="manager",namespace="release-service",source_cluster=~"stone-prd-rh01|stone-prod-p02"}) == 0
+      for: 10m
+      labels:
+        severity: high
+      annotations:
+        summary: >-
+          The Release Service Controller Manager is down in the {{ $labels.source_cluster }} for the last 10 minutes
+        description: >-
+          The Release Service Controller Manager is down in the {{ $labels.source_cluster }} for the last 10 minutes
+        alert_route_key: "{{ $labels.namespace }}"
```

Comment:
The correct key is `alert_routing_key` as you can see in the following examples:
https://github.com/redhat-appstudio/o11y/blob/main/rhobs/alerting/data_plane/prometheus.pod_alerts.yaml#L50
https://github.com/redhat-appstudio/o11y/blob/main/rhobs/alerting/data_plane/prometheus.smee_deployment_alerts.yaml#L22

Reply:
ah ok. Fixed.